client/state/index.js

```diff
@@ -119,6 +121,11 @@ if ( typeof window === 'object' ) {
 		require( './analytics/middleware.js' ).analyticsMiddleware,
 		require( './data-layer/wpcom-api-middleware.js' ).default,
 	);
+
+	//TEMPORARY AT FLOW FIX, NOT INTENDED FOR PROD
+	if ( config.isEnabled( 'automated-transfer' ) && config( 'env' ) !== 'production' ) {
```

Comment:
I can rebase if that lands first. This enhancer just needs to be after sitesSync.